core/test/Test/Pos/Core/AddressSpec.hs

```diff
@@ -70,13 +97,22 @@ spec = describe "Address" $ do
                 esk <- mkEncSecretUnsafe passphrase sk
                 genHDAddrBoot' passphrase esk <$> arbitrary <*> arbitrary
 
+        let hdAddrBootSize = 76 + networkMagicExtraBytes pm
         largestAddressProp "HD address with BootstrapEra distribution"
-            genHDAddrBoot (largestHDAddressBoot fixedNM) 76
-
-pkAndHdwAreShownDifferently :: Bool -> PublicKey -> Bool
-pkAndHdwAreShownDifferently isBootstrap pk =
-    show (makePubKeyAddress fixedNM (IsBootstrapEraAddr isBootstrap) pk) /=
-    (show @Text (makePubKeyHdwAddress fixedNM (IsBootstrapEraAddr isBootstrap)
+            genHDAddrBoot (largestHDAddressBoot nm) hdAddrBootSize
+
+networkMagicExtraBytes :: ProtocolMagic -> Byte
+networkMagicExtraBytes pm = case makeNetworkMagic pm of
+    NMNothing -> 0
+    -- Encoding size:
+    -- Map key: 2 bytes (1 for header, 1 for Word8)
+    -- Map val: 1-5 bytes (1 for header, 0-4 for Word32)
```

Comment:
Is this an `Int32` now? Stupid compiler doesn't know how to type check comments :)

Reply:
Ugh, fixed 😆

Reply:
Ah that one's my fault. oops.